lib/storage/s3.js

```diff
@@ -59,7 +98,22 @@ module.exports = function() {
       }
       https = options.https;
       cachingTime = options.cachingTime;
-      return callback(null);
+      // CORS
+      client.getBucketCors(getCorsConfig, function (err, data) {
```

Comment:
Always use a return statement for clarity when invoking an async API and doing nothing further until the callback is invoked.